src/components/app/details/cIDetails/CIDetails.tsx

```diff
@@ -1129,6 +1129,8 @@ const SecurityTab: React.FC<{ triggerHistory: History }> = (props) => {
         isError: false,
     })
     const { appId } = useParams<{ appId: string }>()
+    const ciPipelineId = props?.triggerHistory?.ciPipelineId
```

Comment:
rather initializing `ciPipelineId` globally. We must move it into `redirectToCreate`

Reply:
Done